index.bs

```diff
@@ -166,6 +166,37 @@ The {{isPresenting}} attribute MUST return the {{VRDisplay}}'s presentation stat
 <dfn attribute for="VRDisplay">capabilities</dfn>
 The {{capabilities}} attribute MUST return the {{VRDisplay}}'s {{VRDisplayCapabilities}} object, a dictionary of capabilities describing the {{VRDisplay}}.
 
+<dfn method for="VRDisplay">getEyeParameters()</dfn>
+Returns the current {{VREyeParameters}} for the given eye. The eye parameters MAY change at any time due to external factors, such as the user changing the IPD with hardware controls. As a result, it is recommended that these values be queried for each frame rather than cached.
+
+<dfn method for="VRDisplay">getPose()</dfn>
+Returns a {{VRPose}} describing the position, orientation, and acceleration of the {{VRDisplay}} that should be used when rendering the next frame of a scene. The User Agent MAY optionally use predictive techniques to estimate what the pose will be at the time that the next frame will be displayed to the user. The pose returned MUST NOT change until JavaScript has returned control to the browser.
+
+<dfn method for="VRDisplay">getImmediatePose()</dfn>
+Returns a {{VRPose}} describing the position, orientation, and acceleration of the {{VRDisplay}} at the time when the call was made. The User Agent SHOULD NOT use predictive techniques when constructing the pose.
+
+<dfn method for="VRDisplay">resetPose()</dfn>
+Reset the pose for the display, treating its current position and orientation as the "origin/zero" values. Future poses returned from {{getPose()}} and {{getImmediatePose()}} will describe positions relative to the {{VRDisplay}}'s position when {{resetPose()}} was last called and will treat the display's yaw when {{resetPose()}} was last called as the forward orientation. The {{VRDisplay}}'s reported roll and pitch do not change when {{resetPose()}} is called as they are relative to gravity. Calling {{resetPose()}} may change the {{sittingToStandingTransform}} matrix of the {{VRStageParameters}}.
+
+<dfn method for="VRDisplay">requestAnimationFrame()</dfn>
+Functionally equivalent to <a href="https://www.w3.org/TR/html51/webappapis.html#animation-frames">window.requestAnimationFrame</a> when the {{VRDisplay}} is not presenting. When the {{VRDisplay}} is presenting the callback is called at the native refresh rate of the {{VRDisplay}}.
```

Comment:
could define the link [above](https://github.com/MozVR/webvr-spec/pull/19/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR17)